.changeset/fresh-schools-fly.md

```diff
@@ -0,0 +1,11 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab': major
```

Comment:
```suggestion
'@backstage/plugin-catalog-backend-module-gitlab': minor
```

Major releases of packages is only something we do in rare cases and that will bring the package up to a stricter versioning policy. This package is on version 0.x which means breaking changes can occur in minor releases.